arrow_test.go

```diff
@@ -95,7 +97,12 @@ func TestArrowBigFloat(t *testing.T) {
 
 func TestArrowIntPrecision(t *testing.T) {
 	db := openDB(t)
-	dbt := &DBTest{t, db}
```

Comment:
What is the reason we are no longer using `DBTest` in some tests? I think that it would be nice to unify the approach and select only one of the options - `DBTest` or `sql.DB`.

Reply:
I wanted to remove `db` from `DBTest` and provide `conn` only. I'd say that using db is only for specific use cases, when default connection cannot be used.